### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by RV-Adrian**
@ghislaineguerin Project doc here:
https://docs.google.com/document/d/1loIBZsGcqwFziAt5a_DtDPHswZPeu5hrk_SMxt5oIvQ/edit#
@brianloosbrock FYI

### CODE REVIEW

**Comment by brianloosbrock** [comment]
@ghislaineguerin Please confirm your good with this once you've reviewed. Thanks.

File: src/ui-components/shopping-cart.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+// SPACING FOR SHOPPING CART
+.shopping-cart {
+
+		.inner-box {
```
Comment (ghislaineguerin) [inline src/ui-components/shopping-cart.scss]: @RV-Adrian Have you seen the .content-box class we use everywhere? It's similar to this style. Not sure if we can reuse or extend? 
.content-box has a .content-box-body with standard padding. Let me know if this box has any special considerations. I'd rather have us adapt content box to work with your prototype, but not limit you in any way. Thanks!

**Comment by ghislaineguerin** [comment]
@RV-Adrian this is what it looks like with content-box class
http://screencast.com/t/9Kdxx9zGj3B5

File: src/ui-components/shopping-cart.scss (hunk 1 above)
Comment (RV-Adrian) [inline src/ui-components/shopping-cart.scss]: @ghislaineguerin  Makes sense.  We're trying to target the spacing only in the shopping cart with this, and it currently uses the 'inner-box' class exclusively from what I can see. 

Would you recommend I stick with .inner-box and styling that only, or try to change the DOM to use .content-box?  (thinking the latter could get tricky)

**Comment by RV-Adrian** [comment]
@ghislaineguerin Ok! Can you msg me that a code snippet so I can check it out?

**Comment by ghislaineguerin** [comment]
@RV-Adrian http://codepen.io/ghislaineguerin/pen/vOKBmq

**Comment by ghislaineguerin** [comment]
@RV-Adrian is inner box an existing style? I couldn't find it anywhere. If it is not, then let's use content-box
Searching 12048 files for ".inner-box"
0 matches

**Comment by RV-Adrian** [comment]
@ghislaineguerin It appears only in the DOM when the cart loads. No existing styles attached to it, but I used it as a hook, since it's there.
<div class="inner-box">
                                <div class="row">
                                    <div class="col-xs-5">
                                        <h2 class="bill-to">Bill To</h2>
...
</div>

**Comment by ghislaineguerin** [comment]
@RV-Adrian got it! then just remove the border styles, radius and extend on .content-box. So we keep the same style. Thanks!

**Comment by RV-Adrian** [comment]
@ghislaineguerin Gotcha, updating now.

**Comment by brianloosbrock** [comment]
@ghislaineguerin @RV-Adrian Are we good now? Thanks

**Comment by RV-Adrian** [comment]
I think so.  How about we use a :+1: as the final signoff ( in this case, from @ghislaineguerin ) ?
(yes, I stole this from devs)

**Comment by ghislaineguerin** [comment]
:+1:

**Comment by brianloosbrock** [comment]
@RV-Adrian @ghislaineguerin Perfect re. final sign off. Thanks!

**Comment by ghislaineguerin** [comment]
@RV-Adrian make sure this branch stays up to date locally so you won't have any issues when rebasing.

**Comment by RV-Adrian** [comment]
@ghislaineguerin Will do. How often do you recommend a Git-Pull? And would rebasing happen as a rule without commits? Lets chat more 'bout it at 1pm!

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Pull Request for v2 of 'In App Purchase' task.
Trello: https://trello.com/c/M4CjeCm7/458-in-app-purchase-improvements-2nd-iteration
UX-Prototype repo: https://github.com/Rise-Vision/ux-prototypes/tree/feature/storage-purchase-action
S3: https://s3.amazonaws.com/ux-design/storage-purchase/prototypes/storage-purchase-action/cart/cart.html

The prior 'v2' branch has been deleted, and all the updated code is now committed here.  Thanks!
@brianloosbrock FYI

**Comment by ghislaineguerin** [comment]
@RV-Adrian It's not letting me merge. Did you rebase?

File: src/ui-components/shopping-cart.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  }
+  .alert-info
+  {
+    margin-bottom: 5px;
+  }
+}
+
+.form-group
+{
+  margin-bottom: 2px;
```
Comment (ghislaineguerin) [inline src/ui-components/shopping-cart.scss]: @RV-Adrian are this classes general? Or only work inside shopping cart?

Comment (RV-Adrian) [inline src/ui-components/shopping-cart.scss]: @ghislaineguerin Shopping cart only, to keep that button above the bottom of the modal.

**Comment by RV-Adrian** [comment]
@ghislaineguerin Regarding rebase, same issue as yesterday where it tells me it's all up-to-date.  Should I try it from a new local branch...?

**Comment by ghislaineguerin** [comment]
@RV-Adrian no worries for now, Brian explained that this is not ready to merge

File: src/ui-components/shopping-cart.scss (hunk 2 above)
Comment (ghislaineguerin) [inline src/ui-components/shopping-cart.scss]: @RV-Adrian We have various utility classes you might want to check out.
.remove-bottom, .remove-top, half-bottom, add-bottom etc
for 0px, 5px and 20px margins. I see many instances of your styles that could benefit from those.

**Comment by brianloosbrock** [comment]
@RV-Adrian We should be rebasing the branch between iterations. Can you figure out what the issue is?

**Comment by RV-Adrian** [comment]
@brianloosbrock For sure, proper rebasing is my 'weekend side project'. Beyond that (and @ghislaineguerin class suggestions above), should we good for this card for today? (ie. no merge)

**Comment by brianloosbrock** [comment]
@RV-Adrian Yes, we're good. No merge. But you should come to a conclusion one way or another on Gishlaines suggestions and make sure to complete the rebase. Thanks.

**Comment by RV-Adrian** [comment]
@ghislaineguerin Finally merge-able!
I had to skip the rebase and use 'merge master' due to changes already being pushed to the remote.  That's the only thing that worked for me, anyway.  Let me know if there's anymore issues?
@brianloosbrock FYI

**Comment by ghislaineguerin** [comment]
@RV-Adrian perfect. Thanks! no other issues

**Comment by RV-Adrian** [comment]
@ghislaineguerin awesome! thanks

**Comment by ghislaineguerin** [comment]
@RV-Adrian is this PR still active?

**Comment by RV-Adrian** [comment]
@ghislaineguerin No, shouldn't this have resolved upon Merge...?

**Comment by ghislaineguerin** [comment]
@RV-Adrian still shows as open?

**Comment by brianloosbrock** [comment]
@ghislaineguerin @RV-Adrian Given these changes are pending and go with the in-app purchase improvements project shouldn't this remain open until they are ready to be implemented? Thanks

**Comment by RV-Adrian** [comment]
@brianloosbrock Good point.. I'm actually not sure. I don't think we've gone to this level of detail. Does keeping the PR in Open status give some value to the dev when they are implementing?  If so, it would be up to Oleg here to give the green light, and then we'd close it.
@ghislaineguerin FYI

**Comment by olegrise** [comment]
@brianloosbrock @ghislaineguerin @adriandd I don't see any value in keeping this PR open. Feel free to merge it.

**Comment by ghislaineguerin** [comment]
@RV-Adrian can this branch be deleted?

**Comment by olegrise** [comment]
Deleting old branches... I like it ;-)

**Comment by RV-Adrian** [comment]
@ghislaineguerin @olegrise Me too! But not this one (yet). :)
I was going to apply these after Coupons is totally solid (likely tomorrow)

**Pull request closed without merging**